lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1194,6 +1194,23 @@ def embedTTFType42(font, characters, descriptor):
             wObject = self.reserveObject('Type 0 widths')
             toUnicodeMapObject = self.reserveObject('ToUnicode map')
 
+            _log.debug(
+                "SUBSET %s characters: %s",
+                filename, "".join(chr(c) for c in characters)
+            )
+            fontdata = _backend_pdf_ps.get_glyphs_subset(
+                filename, "".join(chr(c) for c in characters)
+            )
+            _log.debug(
+                "SUBSET %s %d ↦ %d", filename,
```

Comment:
probably better to stick to ascii in the logs just in case someone is printing these to an old-style windows cmd.exe which may not like that.